src/src/models/FieldModel.ts

```diff
@@ -0,0 +1,37 @@
+import MessageModel from "./MessageModel";
+import OptionModel from "./OptionModel";
+
+interface FieldModel {
+  // Core
+  propertyName: string;
+  title: string;
+  vueType: any;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
  vueType: unknown;
```